SWI-cpp2.h

```diff
@@ -529,13 +536,14 @@ class PlTerm_atom : public PlTerm
 class PlTerm_var : public PlTerm
 {
 public:
-  explicit PlTerm_var() : PlTerm() {}
+  explicit PlTerm_var() { } // PlTerm() calls Pl_new_term_ref()
```

Comment:
We need to call Pl_new_term_ref().   Do I understand correctly that the old implementation called it twice?

Reply:
No - the old code was explicitly "calling" the parent constructor (`PlTerm()`), which in turn "called" the `WrappedC<term_t>` constructor. It is now implicit.